### ISSUE TITLE
Subscript is not announced when using finnish version of nvda, windows 10 and ms word 2019

### ISSUE THREAD

**Issue by burmancomp** [nvaccess/nvda#16194]
### Steps to reproduce:
- open nvda (language finnish)
- open finnish ms word
- ms support suggest shortcut for superscript ctrl+shift+acute accent (ctrl+shift+key next from backspace to left)
- for subscript shortcut should be ctrl+= (ctrl+shift+0)

### Actual behavior:

If nvda language is finnish and keylayout is finnish, superscript shortcut seems to toggle subscript but nothing is spoken. Subscript shortcut speaks current state of subscript, superscript or base if those both are off but it changes nothing. ctrl+plus (+) toggles superscript and speaks it correctly.

Part of problem may be that there is error in finnish shortcut list [https://support.microsoft.com/fi-fi/office/wordin-pikan%C3%A4pp%C3%A4imet-95ef89dd-7142-4b50-afb2-f762f663ceb2](https://support.microsoft.com/fi-fi/office/wordin-pikan%C3%A4pp%C3%A4imet-95ef89dd-7142-4b50-afb2-f762f663ceb2) but there may be something else as well.
#### NVDA installed/portable/running from source:
installed from source with some additional braille related code which should not have effect on this
#### Windows version:
windows 10 22h2 64-bit finnish
#### Name and version of other software in use when reproducing the issue:
office 2019 finnish
#### If NVDA add-ons are disabled, is your problem still occurring?
yes

**Comment by CyrilleB79** [comment]
Hi @burmancomp 


First of all, NVDA is only able to report formatting shortcuts if your keyboard layout, your Office language and your NVDA language are all the same. Using a mix between English and Finnish leads to unexpected results as expected.

Second, you need to check yourself by trying which are the formatting shortcuts that do really work in Finnish Office. Unfortunately, MS documentation is not always accurate nor complete in other languages than English.

If there is an issue specifically with Finnish, you may first write it here, but at last, the fix will have to be done by Finish translators, at least in the current state of the translation framework; it may change in the future.

Note also that in some rare cases, the shortcuts may differ between Office versions and translators have to find the best trade-off.

**Comment by burmancomp** [comment]
Ctrl+plus (+) works for superscript. It both announces it correctly and changes state. But as to subscript: ctrl+shift+plus (+) changes state but nothing is spoken. This happens with word 2019, windows 10 22h2 and nvda (running currently 2023.3.3; different laptop now). Windows, office and nvda language is finnish.

I am about to change title of this issue.

**Comment by CyrilleB79** [comment]
I have just sent [this message](https://groups.io/g/nvda-translations/topics) on the translators mailing list. I hope the Finnish translator will fix it soon.

**Comment by burmancomp** [comment]
Thanks @CyrilleB79.

**Comment by jkinnunen** [comment]
I think this has been fixed now in gestures.ini.

**Comment by CyrilleB79** [comment]
Thanks. Closing.
